### PR TITLE
[Perseus Analytics] Plumb through onAnalyticsEvent to mobile keypad

### PR DESCRIPTION
**Pull request by jeremywiebe**
## Summary:

I had missed plumbing the analytics prop through all the mobile keypad areas. So this PR adds them. ~In the case of the LegacyKeypad, I've not bothered to do analytics as that is going away soon!~ Changed my mind: https://github.com/Khan/perseus/pull/705

Issue: LC-950

## Test plan:

`yarn test`
`yarn start` (all stories with mobile keypad still work, and should show the keypad open/close events in the Storybook "Actions" panel)

### CODE REVIEW

File: packages/perseus/src/widgets/__stories__/test-keypad-context-wrapper.tsx [hunk 1]
```diff
@@ -12,6 +13,11 @@ const Footer = (): React.ReactElement => {
                         onElementMounted={setKeypadElement}
                         onDismiss={() => renderer && renderer.blur()}
                         style={styles.keypad}
+                        useV2Keypad={true}
```
Comment (jeremywiebe) [inline packages/perseus/src/widgets/__stories__/test-keypad-context-wrapper.tsx:16]: @handeyeco I hope/think this is ok. I think at this point we want to be testing with the new keypad most places. Let me know what you think.

File: packages/math-input/src/components/keypad-switch.tsx [hunk 2]
```diff
@@ -11,13 +12,17 @@ type Props = {
...
 };
 
 function KeypadSwitch(props: Props) {
     const {useV2Keypad = false, ...rest} = props;
 
     const KeypadComponent = useV2Keypad ? MobileKeypad : LegacyKeypad;
 
+    // Note: Although we pass the "onAnalyticsEvent" to both keypad components,
+    // only the current one uses it. There's no point in instrumenting the
+    // legacy keypad given that it's on its way out the door.
```
Comment (jeremywiebe) [inline packages/math-input/src/components/keypad-switch.tsx:25]: I changed my mind!  I've got a follow-up PR that'll implement the analytics for the legacy keypad and remove this comment!

File: packages/perseus/src/widgets/__stories__/test-keypad-context-wrapper.tsx (hunk 1 above)
Comment (jeremywiebe) [reply packages/perseus/src/widgets/__stories__/test-keypad-context-wrapper.tsx:16]: Said PR: https://github.com/Khan/perseus/pull/705

**Review by SonicScrewdriver — APPROVED** [review]
Thank you for catching this Jeremy! LGTM! :)

**Review by handeyeco — APPROVED** [review]

**Pull request merged**